cmd/lakectl/cmd/root.go

```diff
@@ -105,7 +105,7 @@ func getClient() *api.ClientWithResponses {
 		DieErr(err)
 	}
 	// if no uri to api is set in configuration - set the default
-	if u.Path == "" || u.Path == "/" {
+	if u.Path == "" || u.Path == "/" || !strings.HasSuffix(serverEndpoint, api.BaseURL) {
```

Comment:
I think we should keep helping the user just in case they didn't specified the path.  If there is a specific path I would like to use it.